lib/screens/profile_screen.dart

```diff
@@ -197,23 +198,37 @@ class ProfileScreen extends StatelessWidget {
   }
 
   Future<void> signInWithApple() async {
-    final rawNonce = generateNonce();
-    final nonce = sha256ofString(rawNonce);
-
-    final appleCredential = await SignInWithApple.getAppleIDCredential(
-      scopes: [
-        AppleIDAuthorizationScopes.email,
-        AppleIDAuthorizationScopes.fullName,
-      ],
-      nonce: nonce,
-    );
-    // Create an `OAuthCredential` from the credential returned by Apple.
-    final oauthCredential = OAuthProvider('apple.com').credential(
-      idToken: appleCredential.identityToken,
-      rawNonce: rawNonce,
-    );
+    // Sign in the user with Firebase.
+    if (kIsWeb) {
```

Comment:
Is this check really needed? The [sign_in_with_apple](https://pub.dev/packages/sign_in_with_apple) package seems to support Web, and their [example](https://pub.dev/packages/sign_in_with_apple/versions/3.3.0/example) shows how to support web with the same function call